Assets/Scripts/PlayerMovement.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
+{
+    private Rigidbody2D rb;
+    private float x;
+    private float y;
+    private Vector2 direction;
+    public float speed;

```

Comment:
Order of variable declarations. Probably best in this case to simply go alphabetically.

Reply:
Also going to start putting all of the public variables on the top. I will make this a consistency thing for all of our files.